pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -655,6 +655,57 @@ public void updatePropertiesFailed(ManagedLedgerException exception, Object ctx)
         return future;
     }
 
+    protected CompletableFuture<Void> internalRemovePropertiesAsync(boolean authoritative, String key) {
+        return validateTopicOwnershipAsync(topicName, authoritative)
+                .thenCompose(__ -> validateNamespaceOperationAsync(topicName.getNamespaceObject(),
+                        NamespaceOperation.CREATE_TOPIC))
```

Comment:
`NamespaceOperation.CREATE_TOPIC` seems odd here. You sure it's right?

Reply:
`Properties` are set when topic is created and it need `NamespaceOperation.CREATE_TOPIC` permission, which correspond to `super user` or `tenant administrator`. 
So I think only roles with above permission are allowed to delete properties.

Reply:
Properties can be updated after topic creation, and `validateTopicOperationAsync(topicName, TopicOperation.PRODUCE)` is used to check the permission. I think it should be consistent?

Reply:
I think that the `validateTopicOperationAsync(topicName, TopicOperation.PRODUCE)` is inappropriate in #17238, because any role with the `topic  produce` action can update the topic‘s properties, in fact only tenant administrators and users with higher permissions can update it.
So maybe we should open another pr to adjust this permission of #17238.

Reply:
I think that the `validateTopicOperationAsync(topicName, TopicOperation.PRODUCE)` is inappropriate in #17238, because any role with the `topic  produce` action can update the topic‘s properties, in fact only tenant administrators and users with higher permissions can update it.
So maybe we should open another pr to adjust this permission of #17238. @Jason918 @AnonHxy

Reply:
> So maybe we should open another pr to adjust this permission of #17238. @Jason918 @AnonHxy

@yuruguo Agree, maybe add new permission like `WRITE_METADATA`.